demos/python_demos/bert_question_answering_embedding_demo/bert_question_answering_embedding_demo.py

```diff
@@ -0,0 +1,360 @@
+                      required=True, type=str)
+    args.add_argument("--input_names_emb",
+                      help="Optional. Names for inputs in MODEL_EMB network. For example 'input_ids,attention_mask,token_type_ids','position_ids'",
+                      default='input_ids,attention_mask,token_type_ids,position_ids',
+                      required=False, type=str)
+    args.add_argument("-m_qa","--model_qa",
+                      help="Optional. Path to an .xml file with a trained model to give exact answer",
+                      default = None,
+                      required=False,type=str)
+    args.add_argument("--input_names_qa",
+                      help="Optional. Names for inputs in MODEL_QA network. For example 'input_ids,attention_mask,token_type_ids','position_ids'",
+                      default='input_ids,attention_mask,token_type_ids,position_ids',
+                      required=False, type=str)
+    args.add_argument("--output_names_qa",
+                      help="Optional. Names for outputs in MODEL_QA network. For example 'output_s,output_e'",
+                      default='output_s,output_e',
+                      required=False, type=str)
+    args.add_argument("-a", "--max_answer_token_num",
+                      help="Optional. Maximum number of tokens in exact answer",
+                      default=15,
+                      required=False, type=int)
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU is "
+                           "acceptable. Sample will look for a suitable plugin for device specified. Default value is CPU",
+                      default="CPU",
+                      required=False, type=str)
+    args.add_argument('-c', '--colors', action='store_true',
+                      help="Optional. Nice coloring of the questions/answers. "
+                           "Might not work on some terminals (like Windows* cmd console)")
+    return parser
+
+def main():
+    log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
+    args = build_argparser().parse_args()
+
+    log.info("Creating Inference Engine")
+    ie = IECore()
+
+    #read model to calculate embedding
+    model_xml_emb = args.model_emb
+    model_bin_emb = os.path.splitext(model_xml_emb)[0] + ".bin"
+
+    log.info("Loading embedding network files:\n\t{}\n\t{}".format(model_xml_emb, model_bin_emb))
+    ie_encoder_emb = ie.read_network(model=model_xml_emb, weights=model_bin_emb)
+    input_names_model_emb = list(ie_encoder_emb.input_info.keys())
+    input_names_emb = args.input_names_emb.split(',')
+    log.info("Expected embedding input names: {}".format(input_names_emb))
+    log.info("Network embedding input names: {}".format(input_names_model_emb))
+    # check input names
+    if set(input_names_model_emb) != set(input_names_emb):
+        log.error("Unexpected embedding nework input names")
```

Comment:
```suggestion
        log.error("Unexpected embedding network input names")
```

(same typo occurs in several other places)